libsolidity/codegen/LValue.h

```diff
@@ -171,7 +173,14 @@ class StorageItem: public LValue
 		langutil::SourceLocation const& _location = {},
 		bool _removeReference = true
 	) const override;
+private:
+	static constexpr solidity::evmasm::Instruction m_storeInstruction = isTransient ? solidity::evmasm::Instruction::TSTORE : solidity::evmasm::Instruction::SSTORE;
+	static constexpr solidity::evmasm::Instruction m_loadInstruction = isTransient ? solidity::evmasm::Instruction::TLOAD : solidity::evmasm::Instruction::SLOAD;
```

Comment:
I think it's useful if you can tell at a glance that an identifier is a template parameter - I'd always at least capitalize them. Or if it's static or not - we should be using the `s_` prefix more consistently for this.

Also, you're in the `solidity` namespace so you can drop it to make things more concise.

```suggestion
	static constexpr evmasm::Instruction s_storeInstruction = IsTransient ? evmasm::Instruction::TSTORE : evmasm::Instruction::SSTORE;
	static constexpr evmasm::Instruction s_loadInstruction = IsTransient ? evmasm::Instruction::TLOAD : evmasm::Instruction::SLOAD;
```

Reply:
Done.

Reply:
You still have some lowercase `isTransient` in the .cpp file.